steps/s2_interpret_segments.py

```diff
@@ -45,7 +45,7 @@ def interpret_segments(configuration: Configuration, dataset: Dataset) -> None:
             activation_file = activation_path(f'{image_id}.npz')
             prediction_file = prediction_path(f'{image_id}.npz')
 
-            if exists(activation_file) and exists(prediction_file):
+            if exists(activation_file) or exists(prediction_file):
```

Comment:
why was this change made? If the activation file exists, but the prediction file does not, then no accuracy can be calculated for the clusters / segments and we definitely have to recompute them, right?
So we can only skip an image if _both_ file exists, only one of them is not enough.

Reply:
My bad, for some reason I don't have prediction files on the test set images(I don't want to run the CNN again for the test)